client/components/input/input.css

```diff
@@ -1,3 +1,117 @@
-.unique-input-class {
-  border: 2px solid blue;
+@import '../typography/typography.css';
+
+input, textarea {
```

Comment:
So you opted in styling default html elements directly. It might look like a good idea at first glance: all inputs and textareas are going to be as per design, right?
There is a gotcha however, which you didn't have chance to experience yet. 
So imagine, we decide to add some additional functionality which involves **differently** styled inputs.
Well, what's the problem, right? We could just assign some class to it smth like ".input-version-2" and just style it?
Problem is: it won't be as simple. To style that class we would have to override **each** property you specified below.
That can be a big deal if all you wanted to do is just make border blue, but else behave like standard html input. You'd have to override a lot of code to achieve that. And you might forget override some and that will lead to bugs.

So, in conclusion, it's better to create a class name here which you'd assign to inputs and textareas instead of changing default behaviour.

This applies also to `fieldset` below.